### PR TITLE
Bug 1275506 - Allow users to click monopod donut charts on overview

### PR DESCRIPTION
**Pull request by spadgett**
Show browse pods pod filtered to just that pod.

https://bugzilla.redhat.com/show_bug.cgi?id=1275506

### CODE REVIEW

**Comment by spadgett** [comment]
@jwforres please review

**Comment by spadgett** [comment]
@jwforres Please wait on review, I don't think this is correct.

**Comment by spadgett** [comment]
@jwforres OK, please look. I've changed it to link directly to the pod page since we know there's only one.

File: pkg/assets/bindata.go [hunk 1]
```diff
@@ -21395,7 +21401,7 @@ link:function(a, b) {
 var d = function() {
 c(function() {
 var c, d;
-c = d3.select(b[0]).select("text.c3-chart-arcs-title"), c && (d = a.getCenterLabelText(), c.selectAll("*").remove(), d.bigText && !d.smText ? c.text(d.bigText) :(c.insert("tspan").text(d.bigText).classed("donut-title-big-pf", !0).attr("dy", 0).attr("x", 0), c.insert("tspan").text(d.smText).classed("donut-title-small-pf", !0).attr("dy", 20).attr("x", 0)));
+c = b[0].querySelector("text.c3-chart-arcs-title"), c && (d = a.getCenterLabelText(), c.innerHTML = d.bigText && !d.smText ? d.bigText :'<tspan dy="0" x="0" class="donut-title-big-pf">' + d.bigText + '</tspan><tspan dy="20" x="0" class="donut-title-small-pf">' + d.smText + "</tspan>");
```
Comment (jwforres) [inline pkg/assets/bindata.go]: shouldn't have this diff, steve hit the same issue, rebase on latest master, and clean and install your bower deps to pull the patternfly update

**Comment by jwforres** [comment]
one issue with bindata, otherwise LGTM

**Comment by spadgett** [comment]
> one issue with bindata, otherwise LGTM

Thanks, fixed bindata. [merge]

**Comment by spadgett** [comment]
[test]

**Comment by spadgett** [comment]
[test]

**Pull request merged**